MobileWallet/Common/Theme.swift

```diff
@@ -165,7 +167,8 @@ struct Fonts: Loopable {
     let actionButton = UIFont(name: "AvenirLTStd-Heavy", size: 16.0)
 
     //Splash
-    let splashTitleLabel = UIFont(name: "AvenirLTStd-Black", size: 30.0)
+    let splashTitleLabelFont = UIFont(name: "AvenirLTStd-Black", size: 30.0)
```

Comment:
Also here, just have it as `splashTitleLabel`. The developers can see what type it is